README.md

```diff
@@ -1,4 +1,52 @@
 # JanusGraph Ambari plugin
+JanusGraph is a scalable graph database optimized for storing and querying graphs containing hundreds of billions of vertices and edges distributed across a multi-machine cluster. JanusGraph is a transactional database that can support thousands of concurrent users executing complex graph traversals in real time. This plugin is designed to make it easier for existing Ambari users to integrate JanusGraph and manage it as a native service. 
+
+Instructions on how to add JanusGraph to [Apache Ambari](https://ambari.apache.org ) or [Hortonworks Data Platform(HDP)](https://hortonworks.com/products/data-platforms/hdp/) as a service can be found below.
+
+## Requirements
+- This plugin has been tested with Ambari versions 2.5, and 2.6. It is not compatible with Ambari version 2.7.
+- The JanusGraph ambari plugin expects SOLR to be available. You need to install the lucidworks-hdpsearch package or mpack to use janusgraph-ambari. If you wish to use the plugin without Solr please look under the known issues section.
```

Comment:
This should mention `HBase`, `Solr`, and `Spark2` as the required core dependencies for JanusGraph.

It would be helpful to add a link to documentation on how to install the HDP Search management pack.